database.sql

```diff
@@ -35,32 +35,39 @@ CREATE TABLE IF NOT EXISTS `users` (
 ) ENGINE = InnoDB;
 
 CREATE TABLE IF NOT EXISTS `characters` (
-  `charid` int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
-  `userid` int(11) UNSIGNED NOT NULL,
+  `charid` SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `userid` SMALLINT UNSIGNED NOT NULL,
```

Comment:
`SMALLINT` is only 65535, which while enough for most servers I still wouldn't recommend capping it at such a small number; and it makes updating the datatype in the future a nuisance.

Just roll with `INT`.

Reply:
> Reviewed, but I really couldn't care less about supporting MySQL 8; I'm only even using MySQL because it's standard on FiveM and needed in the inventory to support other frameworks.

Understandable, I am probably one in 1000 who use MySQL 8 instead of MariaDB..  but life is life :D